src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -597,6 +597,20 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('dtu', help='TThe total shared DTU for the elastic eool.')
         c.argument('storage_mb', help='Storage limit for the elastic pool in MB.')
 
+    #####
+    #           sql elastic-pool op
+    #####
+    with self.argument_context('sql elastic-pool op') as c:
+        c.argument('elastic_pool_name',
+                   options_list=['--elastic-pool', '-ep'],
```

Comment:
I like the idea of a shortcut for elastic pool name, but let's design & implement this separately. Can you file an issue https://github.com/Azure/azure-cli/issues for us to do this later?

Reply:
And remove `-ep` for now?

Reply:
@jaredmoo removed the short name -ep for now. And created a new issue on that. https://github.com/Azure/azure-cli/issues/5890